### PR TITLE
Sp03 navbar progression

### PR DESCRIPTION
**Pull request by JulienOza**

### CODE REVIEW

**Review by SebSamPro — APPROVED** [review]

**Review by LucasMerino10 — APPROVED** [review]

File: frontend/src/components/Clues/Clues.css [hunk 1]
```diff
@@ -15,6 +16,14 @@
   top: 3rem;
 }
 
+.clue {
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 16px;
+  padding: 1rem;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Clues/Clues.css]: Could be done in one line

File: frontend/src/components/QuizProgression/QuizProgression.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.progression_text {
+  text-align: center;
+  font-family: var(--second-font);
+  font-size: 1.2rem;
+  font-weight: bold;
+  background: var(--button-color);
+  border: var(--button-border);
+  border-radius: 16px;
+  padding: 1rem;
```
Comment (Yoann-Fortin) [inline frontend/src/components/QuizProgression/QuizProgression.css]: Could be done in one line

File: frontend/src/components/poster/Poster.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ function Poster({ poster }) {
   const imageUrl = "https://image.tmdb.org/t/p/original";
   return (
     <img
-      className="poster__img"
+      className="poster__img poster poster"
```
Comment (Yoann-Fortin) [inline frontend/src/components/poster/Poster.jsx]: Twice poster

File: frontend/src/components/Clues/Clues.css (hunk 1 above)
Comment (JulienOza) [reply frontend/src/components/Clues/Clues.css]: Changed

File: frontend/src/components/QuizProgression/QuizProgression.css (hunk 2 above)
Comment (JulienOza) [reply frontend/src/components/QuizProgression/QuizProgression.css]: Changed

File: frontend/src/components/poster/Poster.jsx (hunk 3 above)
Comment (JulienOza) [reply frontend/src/components/poster/Poster.jsx]: argh, it's some test residue ! Removed !

**Pull request merged**